Remove unused `release.jks` from `.configure`

wzieba opened the pull request:
### Description
<!-- Take the time to write a good summary. Why is it needed? What does it do? When fixing bugs try to avoid just writing “See original issue” – clarify what the problem was and how you’ve fixed it. -->
This file is not used anywhere in the project. It's a leftover, from times when `configure` tool was copying files from local `.mobile-secrets` repository, instead of decrypting them from `.enc` files (hence: `encrypt: false`)

### Testing information
<!-- This is your opportunity to break out individual scenarios that need testing (when necessary) and/or include a checklist for the reviewer to go through. Consider documenting the following from your own completed testing: devices used, alternate workflows, edge cases, affected areas, critical flows, areas not tested, and any remaining unknowns. Provide feedback on this new section of the PR template through Sept 30, 2024 to Apps Quality; additional context here: https://woomobilep2.wordpress.com/2024/05/06/woocommerce-mobile-quality-report-march-april/#comment-12036 -->
Double check that `storeFile`, `storePassword` _properties_ or `release.jks` file are not used anywhere in the repository.

## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:

- [ ] The PR is small and has a clear, single focus, or a valid explanation is provided in the description. If needed, please request to split it into smaller PRs.
- [ ] Ensure Adequate Unit Test Coverage: The changes are reasonably covered by unit tests or an explanation is provided in the PR description.
- [ ] Manual Testing: The author listed all the tests they ran, including smoke tests when needed (e.g., for refactorings). The reviewer confirmed that the PR works as expected on big (tablet) and small (phone) in case of UI changes, and no regressions are added.

<!-- Pull request guidelines: https://github.com/woocommerce/woocommerce-android/blob/develop/docs/pull-request-guidelines.md -->

wzieba committed:
Remove unused `release.jks` from `.configure`

This file is not used anywhere in the project. It's a leftover, from times when `configure` tool was copying files from local `.mobile-secrets` repository, instead of decrypting them from `.enc` files (hence: `encrypt: false`)

dangermattic commented:
<!--
  0 Errors
  0 Warnings
  1 Message: This PR is still a Draft: some...
  0 Markdowns
-->
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">This PR is still a Draft: some checks will be skipped.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

wzieba committed:
Update secrets to not contain properties about `release.jks` file

ParaskP7 reviewed (approved):
👋 @wzieba !

I have reviewed and tested this PR as per the instructions, everything LGTM, thanks fro the clean-up! 🌟

> Double check that storeFile, storePassword properties or release.jks file are not used anywhere in the repository.

I did run a quick search on all these 3 and I actually found a match for both, `storeFile` and `storePassword`, do check this [tools/install-app-bundle.sh](https://github.com/woocommerce/woocommerce-android/blob/60fc8b4ecd635b7a068da7008a1de025773fbf52/tools/install-app-bundle.sh#L20C69-L21) script.

FYI: I don't think this script is used too much, might even be out-dated, but let's double check that anyway, wdyt? 🤔

PS: No matter, I am approving as to not block you on this. ✅

ParaskP7 commented:
Btw, do you think we should do the same on the JP/WPAndroid too ([.configure](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/.configure#L32-L34))? 🤔

wzieba committed:
Remove unused release-related scripts

Initially added in https://github.com/woocommerce/woocommerce-android/pull/1028

ParaskP7 commented on tools/rename_apk_aab.sh:
👋 @wzieba and let's consult with @AliSoftware if this script is/could be still useful, that is, post Release V2 migration, maybe it is indeed not that useful anymore. 🤷 

PS: Same applies to the JP/WPAndroid related such script ([rename_apk_aab.sh](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/tools/rename_apk_aab.sh)), which is what I actually used to copy-paste this script for WCAndroid.

AliSoftware replied:
Yeah I don't think it's that useful anymore, especially because we now automate the download of the universal signed APK from Google Play to attach it to the GitHub Release now, so this process is no longer manual nor requiring renaming the files.

ParaskP7 replied:
Great, yea, I thought I remember someone somewhere mentioned the automation of the download, thanks for the clarification/confirmation on that Olivier! 🙇

AliSoftware replied:
For refs: https://github.com/woocommerce/woocommerce-android/blob/48cb55e80e298ae52c9777af81f19aeba2e2d78a/fastlane/Fastfile#L1133-L1143

wzieba replied:
I'd also assume that our release software doesn't rely on scripts stored locally in repository, but rather uses its own implementations or other tools like release-toolkit

wzieba commented:
> Btw, do you think we should do the same on the JP/WPAndroid too ([.configure](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/.configure#L32-L34))? 🤔

Sure @ParaskP7 , will do! Thanks for the review, merging :shipit:
